### DISCUSSION TITLE
Support HiDPI so user can see what is going on [I may have filed this already; my previous request seems lost]

### DISCUSSION THREAD

**Discussion by LinuxOnTheDesktop** [fwupd/fwupd#3233] [Ideas]
On HiDPI screens the messages about 'capsules' etc. are hard to read. This becomes a serious problem when there is a problem with an update - for it hinders attempts to see what the problem is.

So: make the text larger, at least on HiDPI screens.

**Comment by hughsie** [comment]
I don't know how; I thought in EFI mode you just tell the console what to print and it does it? @superm1 might know more.

**Reply by superm1** [reply]
Matches my expectations. You don't get to switch resolutions from the gop driver.

**Comment by LinuxOnTheDesktop** [comment]
Seemingly then there is a problem here - the text is too small - that cannot be solved by changing resolution or by changing some sort of scaling setting. OK. What about simple font size? Is that not changeable either?

**Reply by superm1** [reply]
I'm not aware of any way to do this with text mode gop either.

Btw I expect the same experience from EFI shell.

**Comment by LinuxOnTheDesktop** [comment]
[This over at systemd](https://github.com/systemd/systemd/issues/2750) seems a relevant (if long and perhaps inconclusive) discussion.
